ffv1.md

```diff
@@ -0,0 +1,960 @@
+
+__a < b__       means a less than b.
+
+__a <= b__      means a less than or equal b.
+
+__a == b__      means a equal to b.
+
+__a != b__      means a not equalto b.
+
+__a && b__      means boolean logical “and” of a and b.
+
+__a || b__      means boolean logical “or” of a and b.
+
+__!a__          means boolean logical “not”.
+
+__a ? b : c__   means b if a is true otherwise c.
+--------------- ----------------------------------------------------------------
+
+## Order of operation precedence
+
+When order of precedence is not indicated explicitly by use of parentheses, operations are evaluated in the following order (from top to bottom, operations of same precedence being evaluated from left to right):
+
+|                                          |
+|:----------------------------------------:|
+|<code>a++, a–</code>                      |
+|<code>!a, -a</code>                       |
+|<code>a * b, a / b, a % b </code>         |
+|<code>a + b, a - b</code>                 |
+|<code>a << b, a >> b</code>               |
+|<code>a < b, a <= b, a > b, a >= b</code> |
+|<code>a == b, a != b</code>               |
+|<code>a & b</code>                        |
+|<code>a &#124; b</code>                   |
+|<code>a && b</code>                       |
+|<code>a &#124;&#124; b</code>             |
+|<code>a ? b : c</code>                    |
+|<code>a = b, a += b, a -= b</code>        |
+
+## Range
+
+__a...b__ means any value starting from a to b, inclusive.
+
+## Bitstream functions
+
+**remaing_bits_in_bitstream( )** means the count of remaining bits after the current position in the bitstream. It is computed from the NumBytes value multiplied by 8 minus the count of bits already read by the bitstream parser.
+
+# General Description
+
+Each frame is split in 1 to 4 planes (Y, Cb, Cr, Alpha). In the case of the normal YCbCr colorspace the Y plane is coded first followed by the Cb and Cr planes, if an Alpha/transparency plane exists, it is coded last. In the case of the JPEG2000-RCT colorspace the lines are interleaved to improve caching efficiency since it is most likely that the RCT will immediately be converted to RGB during decoding; the interleaved coding order is also Y,Cb,Cr,Alpha.
+
+Samples within a plane are coded in raster scan order (left->right, top->bottom). Each sample is predicted by the median predictor from samples in the same plane and the difference is stored see [Coding of the Sample Difference](#coding-of-the-sample-difference).
+
+## Border
+
+For the purpose of the predictior and context, samples above the coded slice are assumed to be 0; samples to the right of the coded slice are identical to the closest left sample; samples to the left of the coded slice are identical to the top right sample (if there is one), otherwise 0.
+
+| | | | | | | | |
+|---|---|---|---|---|---|---|---|
+|0|0| |0|0|0| |0|
+|0|0| |0|0|0| |0|
+|0|0| |a|b|c| |c|
+|0|a| |d| |e| |e|
+|0|d| |f|g|h| |h|
```

Comment:
Some pretty printing please. Also what's up with the empty table header row? Is there a way to make it disappear?